module/vision/VisualMesh/README.md

```diff
@@ -3,12 +3,32 @@ VisualMesh
 
 ## Description
 
+Runs the Visual Mesh over a given image and emits the result.
 
 ## Usage
 
+This module requires on the following configuration files to be set:
+- `data/config/networks` should include a `yaml` file with Visual Mesh network configuration, which can be obtained from tools in the [Visual Mesh](https://github.com/Fastcode/VisualMesh) repository after training. A guide on updating this can be found on [NUbook](https://nubook.nubots.net/guides/main/maintaining-subsystems#vision).
+- `data/config/VisualMesh.yaml` should include information on what cameras and network configuration to use.
+
+Once these are set appropriately, this module will trigger on any `message::input::Image`s emitted in the system.
+
+The module uses the `./visualmesh/VisualMeshRunner` to run the Visual Mesh.
+
+`VisualMeshRunner` gets a `LoadedModel` from `./visualmesh/load_model`, which uses the network configuration specified in `./data/config/VisualMesh.yaml` to create a network model through the `VisualMesh` library. This `LoadedModel` is the Visual Mesh model that the `Image` will be run on, and is only updated if the configuration changes.
+
+This `LoadedModel` is then used in the `VisualMeshRunner` anytime an `Image` needs to be processed. The `VisualMeshRunner` takes the `Image` and `Hcw` (world to camera homogeneous transformation), gets the lens and height data from these and passes it through to the `VisualMesh` itself to get the results. The `VisualMeshRunner` then returns the results and the module emits those results as a `message::vision::VisualMesh` message. This will rerun with a new `Image` and `Hcw` every time a new `Image` is received.
```

Comment:
I suggest adding a link to the NUbook Mathematics page over the (world to camera homogenous transformation) section. Just found that the resources there were quite useful when I was getting up to speed for behaviour:

```suggestion
This `LoadedModel` is then used in the `VisualMeshRunner` anytime an `Image` needs to be processed. The `VisualMeshRunner` takes the `Image` and `Hcw` [(world to camera homogenous transformation)](https://nubook.nubots.net/system/foundations/mathematics#homogeneous-transformations)
, gets the lens and height data from these and passes it through to the `VisualMesh` itself to get the results. The `VisualMeshRunner` then returns the results and the module emits those results as a `message::vision::VisualMesh` message. This will rerun with a new `Image` and `Hcw` every time a new `Image` is received.
```